inspirehep/modules/orcid/tasks.py

```diff
@@ -27,15 +27,15 @@
 from flask import current_app
 from celery import shared_task
 from celery.utils.log import get_task_logger
-from redis import StrictRedis
-from redis_lock import Lock
 from simplejson import loads
 from invenio_oauthclient.utils import oauth_get_user, oauth_link_external_id
 from invenio_oauthclient.models import RemoteToken, User, RemoteAccount, UserIdentity
 from invenio_db import db
 from inspire_utils.record import get_value
 from invenio_oauthclient.errors import AlreadyLinkedError
 
+from inspirehep.modules.cache.providers.redis_client import redis_locked_context
```

Comment:
Mh, it sounds to me that this is a cache util, not a new provider, so I'd put it in `inspirehep.modules.cache.utils`.

Reply:
sounds better :+1: